lib/exercism/team_stream_filters.rb

```diff
@@ -72,7 +72,7 @@ def sql
     # rubocop:disable Metrics/MethodLength
     def views_sql
       <<-SQL
-      SELECT 1 AS id, COUNT(views.id) AS total
+      SELECT 'team_stream' AS id, COUNT(1) AS total
```

Comment:
@kytrinyx I think we can just hardcode the `id` for an aggregated submission count since the stream name is not important for the query. What do you think?